Objects/dictobject.c

```diff
@@ -464,6 +464,12 @@ estimate_keysize(Py_ssize_t n)
         (d)->ma_keys->dk_lookup = lookdict_unicode; \
     }
 
+#define MAYBE_NONUNICODE(d, key) \
```

Comment:
This is only used twice, and it is not clear from the name what it does.
Just replace its uses with the expansion.

Reply:
Done.